src/utils/streams/index.ts

```diff
@@ -31,6 +33,16 @@ const decodeStream = async (stream: ReadableStream) => {
     }
 
     result.push(...lines);
+
+    if (lineSizeLimit) {
+      if (result[result.length - 1].length > lineSizeLimit) {
```

Comment:
For testing purposes I set the `lineSizeLimit` to 10 bytes and this line was really flaky, so the `trimmedLines` flag was only sometimes set. This seems like it is checking if the 2nd to last line of the log is too long? Also, what is the difference between the trim operation performed here and calling `trimLineToMaxSize` in `src/context/LogContext/state.ts`?

Reply:
Looks like the commit history on this pr is pretty messed up. I'm gonna close it and opened https://github.com/evergreen-ci/parsley/pull/342 instead.